crowbar_framework/config/routes.rb

```diff
@@ -24,102 +24,103 @@
+  get "utils/:controller/1.0/export(.:format)", action: "export", as: "utils_export"
+  get "utils/:controller/1.0(.:format)", action: "utils", as: "utils_barclamp"
+  get "utils/import/:id(.:format)", controller: "support", action: "import", constraints: { id: /[^\/]+/ }, as: "utils_import"
+  get "utils/upload/:id(.:format)", controller: "support", action: "upload", constraints: { id: /[^\/]+/ }, as: "utils_upload"
+  get "utils/repositories(.:format)", controller: "repositories", action: "index", as: "repositories"
+  post "utils/repositories/sync(.:format)", controller: "repositories", action: "sync", as: "sync_repositories"
+  post "utils/repositories/activate(.:format)", controller: "repositories", action: "activate", as: "activate_repository"
+  post "utils/repositories/deactivate(.:format)", controller: "repositories", action: "deactivate", as: "deactivate_repository"
+  post "utils/repositories/activate_all(.:format)", controller: "repositories", action: "activate_all", as: "activate_all_repositories"
+  post "utils/repositories/deactivate_all(.:format)", controller: "repositories", action: "deactivate_all", as: "deactivate_all_repositories"
 
   # barclamps
-  get "crowbar/:controller/1.0/help", action: "help", as: "help_barclamp"
-  get "crowbar/:controller/1.0/proposals/nodes", action: "nodes", as: "barclamp_nodes"
-  put "crowbar/:controller/1.0/proposals", action: "proposal_create", as: "create_proposal_barclamp"
-  get "crowbar/:controller/1.0/proposals", action: "proposals", as: "proposals_barclamp"
-  post "crowbar/:controller/1.0/proposals/commit/:id", action: "proposal_commit", as: "commit_proposal_barclamp"
+  get "crowbar/:controller/1.0/help(.:format)", action: "help", as: "help_barclamp"
+  get "crowbar/:controller/1.0/proposals/nodes(.:format)", action: "nodes", as: "barclamp_nodes"
+  put "crowbar/:controller/1.0/proposals(.:format)", action: "proposal_create", as: "create_proposal_barclamp"
+  get "crowbar/:controller/1.0/proposals(.:format)", action: "proposals", as: "proposals_barclamp"
+  get "crowbar/:controller/1.0/proposals/template(.:format)", action: "proposal_template", as: "template_proposal_barclamp"
+  post "crowbar/:controller/1.0/proposals/commit/:id(.:format)", action: "proposal_commit", as: "commit_proposal_barclamp"
   get "crowbar/:controller/1.0/proposals/status(/:id)(/:name)(.:format)", action: "proposal_status", as: "status_proposals_barclamp"
-  delete "crowbar/:controller/1.0/proposals/:id", action: "proposal_delete", as: "delete_proposal_barclamp"
-  delete "crowbar/:controller/1.0/proposals/dequeue/:id", action: "proposal_dequeue", as: "dequeue_barclamp"
-  post "crowbar/:controller/1.0/proposals/:id", action: "proposal_update", as: "update_proposal_barclamp"
-  get "crowbar/:controller/1.0/proposals/:id", action: "proposal_show", as: "proposal_barclamp"
+  delete "crowbar/:controller/1.0/proposals/:id(.:format)", action: "proposal_delete", as: "delete_proposal_barclamp"
```

Comment:
Metrics/LineLength: Line is too long. [117/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)